chipsec/logger.py

```diff
@@ -117,58 +117,56 @@ class ColorLogger:
         NORMAL = 8
 
         def log_test(self):
-            print ColorLogger.BOLD + "BOLD" + ColorLogger.ENDC
-            print ColorLogger.UNDERLINE + "UNDERLINE" + ColorLogger.ENDC
+            print("{}BOLD{}".format(ColorLogger.BOLD,ColorLogger.ENDC))
+            print("{}UNDERLINE{}".format(ColorLogger.UNDERLINE,ColorLogger.ENDC))
             for color_type in (ColorLogger.LIGHT,ColorLogger.DARK,ColorLogger.BACKGROUND,ColorLogger.LIGHT_BACKGROUND):
                 for code in xrange(ColorLogger.GRAY, ColorLogger.NORMAL+1):
                     self.log(color_type+code, color_type, code )
 
 
         def log(self,msg, color_type=LIGHT,color=8):
-            print ColorLogger.format(msg, color_type, color)
+            print(ColorLogger.format(msg, color_type, color))
 
         @staticmethod
         def format(msg, color_type=LIGHT,color=8):
             return ( '\033[%im%s%s'%(color_type+color,str(msg),ColorLogger.ENDC))
 
-    COLOR_ID = {
-      BLACK  : ColorLogger.NORMAL,
-      RED    : ColorLogger.RED,
-      GREEN  : ColorLogger.GREEN,
-      YELLOW : ColorLogger.YELLOW,
-      BLUE   : ColorLogger.BLUE,
-      MAGENTA: ColorLogger.PURPLE,
-      CYAN   : ColorLogger.CYAN,
-      WHITE  : ColorLogger.WHITE
-      }
-
-    def log_color( fg_color, text ):
-        _text = ColorLogger.format(text, ColorLogger.LIGHT,COLOR_ID[ fg_color ])
-        print _text
```

Comment:
Leave the old color logging in place for now.  This will also require some additional changes in each of the logging interfaces.